content/en/blog/2023/experimental-dual-stack/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: "Support of Dual Stack Kubernetes Clusters"
+description: "Experimental and Alpha support for Dual Stack Kubernetes Clusters."
+publishdate: 2023-002-07T13:00:00-06:00
+attribution: "Steve Zhang (Intel), Alex Xu (Intel), Iris Ding (Intel), Jacob Delgado (F5), Ying-chun Cai (formerly F5)"
+keywords: [dual-stack]
+---
+
+Over the past year, both Intel and F5 have collaborated on an effort to bring support for
+[Kubernetes Dual-Stack networking](https://kubernetes.io/docs/concepts/services-networking/dual-stack/) to Istio.
+
+## Background
+
+The journey has taken us longer than anticipated and we continue to have work to do. Initially started with a design based
+on a reference design implementation from F5. The design lead to an [RFC](https://docs.google.com/document/d/1oT6pmRhOw7AtsldU0-HbfA0zA26j9LYiBD_eepeErsQ/edit?usp=sharing) that
+caused us to re-examine our approach. Notably, there were concerns about memory and performance issues that the community wanted
+to be addressed before implementation.
+
+## Experimental Dual Stack branch
+
+While the team was re-evaluating its options we created a new branch, [experimental-dual-stack]({{< github_raw >}}/tree/experimental-dual-stack),
```

Comment:
I will say that while that branch builds locally, it fails the CI pipeline if another PR is added.